compile-reproducer.sh

```diff
@@ -0,0 +1,65 @@
+#! /bin/bash
+
+usage() {
+        echo "usage: $(basename "$0") [-h|--help] <clients_version> [clients_build]"
+}
+
+if [ $# -lt 1 ] || [[ "$@" = *"-h"* ]]; then
+        usage
+        exit 0
+fi
+
+TARGET_DIR=./dist/tcpkeepalivesettings-reproducer
+REPRODUCER_DIR=./TcpKeepAliveSettings-reproducer/TcpKeepAliveSettings
+CLI_VER=$1
+CLI_BUILD=${2:-""}
+
+if [[ "$CLI_BUILD" != "" ]]; then
+	CLI_BUILD=".$CLI_BUILD"
+fi
+
+# Create directory for finished product
+rm -rf $TARGET_DIR
+mkdir -p $TARGET_DIR
+
+# Search and replace Package AMQPNetLite.Core with AMQP.dll
+sed -i 's/PackageReference Include="AMQPNetLite.Core" Version="2.3.0/Reference Include="AMQP.dll/g' $REPRODUCER_DIR/TcpKeepAliveSettings.csproj
+
+# Download dotnet core client
+if ! wget -q -t 10 "http://download-node-02.eng.bos.redhat.com/devel/candidates/amq/AMQ-CLIENTS-$CLI_VER$CLI_BUILD/amq-clients-$CLI_VER-dotnet-core.zip"; then
```

Comment:
I am not sure if we should have this line here. This repo is public to the whole world and we use our (probably) internal path.

Reply:
Which actually leads me to retorical question: Why am I using sed everytime, why don't I just change it...